annotations/split_manifest_grants.py

```diff
@@ -0,0 +1,63 @@
+"""Split Manifests CSV
+This script will split a manifest csv by grant number and output
+results into individual CSVs.
+author: verena.chung
+author: brynn.zalmanek
+"""
+
+import os
+import argparse
+import pandas as pd
+
+
+def get_args():
+    """Set up command-line interface and get arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("manifest",
+                        type=str,
+                        help="path of manifest to be split")
+    parser.add_argument("manifest_type",
+                        type=str,
+                        choices=["publication", "dataset", "tool", "project"],
+                        help="type of manifest to split, e.g. publicaiton")
+    parser.add_argument("folder",
+                        type=str,
+                        help="folder path to save split manifests in")
+    return parser.parse_args()
+
+
+def get_df(manifest_csv):
+    """Convert manifest to data frame."""
+    df = pd.read_csv(manifest_csv)
+    return (df)
+
+
+def split_manifest(df, type, directory):
```

Comment:
`type` is one of Python's [built-in functions](https://docs.python.org/3/library/functions.html#type), so I'd avoid using this variable name.

Reply:
Ah thanks!